ofctrl/ofSwitch.go

```diff
@@ -158,9 +158,6 @@ func (self *OFSwitch) IsReady() bool {
 
 // Handle switch connected event
 func (self *OFSwitch) switchConnected() error {
-	if err := self.clearGroups(); err != nil {
-		return fmt.Errorf("fails to clear groups: %v", err)
-	}
```

Comment:
Just to confirm, connecting/reconnecting to OVS will no longer make any change to OpenFlow entries, right?

Reply:
That's correct. AFAIK, this is the only place that modified OF entries directly. We inherited that from the original ofnet library from contiv, for reasons that are not clear, and we have kept it as a special case until now. For example, in Antrea, we rely on this behavior, while for meter entries we have custom logic to delete meter entries. I see no reason to treat them differently.

Someone else actually requested that change a while ago, and provided some extra context about contiv: https://github.com/antrea-io/ofnet/pull/9

Reply:
Thanks for the explanation, makes sense to me.